### ISSUE TITLE
Enocean PTM 216Z: missing command

### ISSUE THREAD

**Issue by THorst92** [Koenkk/zigbee2mqtt#13804]
### What happened?

Hello,

I bought two Enocean PTM 216Z and paired them to Zigbee2MQTT via a Philips Hue Smart Plug.
Everything worked so far and they are indicated as paired device. 

<img width="876" alt="Bildschirmfoto 2022-09-01 um 19 51 02" src="https://user-images.githubusercontent.com/22470970/187980190-58e1dce4-1342-44fc-9c40-d916347780bc.png">

But if I press a button always a failure message appears:
Enocean PTM 216Z: missing command 'xxx'

Here the codes:

- Top right press: 105_4_254_165_xxx (xxx e.g. 255 or 242)
- Top left press: 105_1_254_165_xxx
- Bottom right press: 105_8_254_165_xxx
- Bottom left press: 105_2_254_165_xxx
- Both top press: 105_5_254_165_xxx
- Both bottom press: 105_10_254_165_xxx
- Release always: 106_0_254_165_xxx

The last "_xxx" changes and seems to be the link quality or so, because it gets lower if I have a higher distance to the smart plug.

Do you have an idea what is the problem?

### Zigbee2MQTT version

1.27.2

### Adapter firmware version

20220219

### Adapter

SONOFF Zigbee 3.0 USB Dongle Plus (zStack3x0)

### Debug log

```
Debug 2022-09-01 19:45:23Received Zigbee message from '0x00000000xxxxxxxx', type 'commandNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[6,254,165,255],"type":"Buffer"}},"commandID":105,"frameCounter":183,"gppGddLink":165,"gppNwkAddr":65030,"options":21664,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 2948

Error 2022-09-01 19:45:23PTM 216Z: missing command '105_6_254_165_255'

Debug 2022-09-01 19:45:23Received Zigbee message from '0x00000000xxxxxxxx', type 'commandCommisioningNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[6,76,73,140,157,71,80,66],"type":"Buffer"}},"commandID":105,"frameCounter":183,"gppGddLink":73,"gppNwkAddr":19462,"options":2848,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 0

Debug 2022-09-01 19:45:24Received Zigbee message from '0x00000000xxxxxxxx', type 'commandNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[0,254,165,255],"type":"Buffer"}},"commandID":106,"frameCounter":184,"gppGddLink":165,"gppNwkAddr":65024,"options":21664,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 2948

Error 2022-09-01 19:45:24PTM 216Z: missing command '106_0_254_165_255'

Debug 2022-09-01 19:45:24Received Zigbee message from '0x00000000xxxxxxxx', type 'commandCommisioningNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[0,76,73,140,60,220,249,91],"type":"Buffer"}},"commandID":106,"frameCounter":184,"gppGddLink":73,"gppNwkAddr":19456,"options":2848,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 0

Debug 2022-09-01 19:45:26Received Zigbee message from '0x00000000xxxxxxxx', type 'commandNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[6,254,165,255],"type":"Buffer"}},"commandID":105,"frameCounter":185,"gppGddLink":165,"gppNwkAddr":65030,"options":21664,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 2948

Error 2022-09-01 19:45:26PTM 216Z: missing command '105_6_254_165_255'

Debug 2022-09-01 19:45:26Received Zigbee message from '0x00000000xxxxxxxx', type 'commandCommisioningNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[6,76,73,141,154,203,127,48],"type":"Buffer"}},"commandID":105,"frameCounter":185,"gppGddLink":73,"gppNwkAddr":19462,"options":2848,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 0

Debug 2022-09-01 19:45:27Received Zigbee message from '0x00000000xxxxxxxx', type 'commandNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[0,254,165,255],"type":"Buffer"}},"commandID":106,"frameCounter":186,"gppGddLink":165,"gppNwkAddr":65024,"options":21664,"payloadSize":1,"srcID": xxxxxxxxxx}' from endpoint 242 with groupID 2948

Error 2022-09-01 19:45:27PTM 216Z: missing command '106_0_254_165_255'

Debug 2022-09-01 19:45:27Received Zigbee message from '0x00000000xxxxxxxx', type 'commandCommisioningNotification', cluster 'greenPower', data '{"commandFrame":{"raw":{"data":[0,76,73,141,22,232,60,157],"type":"Buffer"}},"commandID":106,"frameCounter":186,"gppGddLink":73,"gppNwkAddr":19456,"options":2848,"payloadSize":1,"srcID":xxxxxxxxxx}' from endpoint 242 with groupID 0
```

**Comment by THorst92** [comment]
One additional hint. The last "_xxx" changes and seems to be the link quality or so, because it gets lower if I have a higher distance to the smart plug.

**Comment by THorst92** [comment]
Fixed with Koenkk/zigbee-herdsman-converters#4621
